content/en/dynamic_instrumentation/symdb/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Symbol Database
+kind: documentation
+is_beta: true
+private: true
+further_reading:
+- link: "/dynamic_instrumentation/"
+  tag: "Documentation"
+  text: "Learn more about the Dynamic Instrumentation"
+---
+
+{{< site-region region="gov,ap1" >}}
+<div class="alert alert-warning">Symbol Database is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+{{< beta-callout-private url="https://forms.gle/UG9EELAy8Li6z2jW8" >}}
+Interested in an improved user experience when creating Dynamic Instrumentation probes? Join the Symbol Database private beta here.
+{{< /beta-callout-private >}}
+
+## Overview
+
+Symbol Database uploads non-sensitive symbols and metadata from your application to the Datadog backend to enhance the user experience of [Dynamic Instrumentation][1] with IDE-like features like search and auto-complete. The uploaded data includes class and method names as well as field-, argument- and local variable names and related meta data like line numbers.
+
+## Getting started
+
+### Prerequisites
+
+Symbol Database requires the following:
+
+- You have on-boarded with [Dynamic Instrumentation][1]
+- [Datadog Agent][2] 7.45.0 or higher is installed alongside your service.
+- [Remote Configuration][3] is enabled in that Agent.
+- For Java applications, tracing library [`dd-trace-java`][6] 1.25.0 or higher.
+- For Python applications, tracing library [`dd-trace-py`][7] 2.6.0 or higher.
+- For .NET applications, tracing library [`dd-trace-dotnet`][8] 2.42.0 or higher. TODO
```

Comment:
```suggestion
- For .NET applications, tracing library [`dd-trace-dotnet`][8] 2.42.0 or higher.
```

Reply:
Are these the future anticipated version were this will be available?

Reply:
yeah, I'm removing python and dotnet for the time being.